src/RestQuery.js

```diff
@@ -69,11 +68,20 @@ function RestQuery(
   // For example, passing an arg of include=foo.bar,foo.baz could lead to
   // this.include = [['foo'], ['foo', 'baz'], ['foo', 'bar']]
   this.include = [];
+  var keysForInclude = [];
 
   // If we have keys, we probably want to force some includes (n-1 level)
   // See issue: https://github.com/parse-community/parse-server/issues/3185
   if (Object.prototype.hasOwnProperty.call(restOptions, 'keys')) {
-    const keysForInclude = restOptions.keys
+    keysForInclude = restOptions.keys;
+  }
+
+  if (Object.prototype.hasOwnProperty.call(restOptions, 'excludeKeys')) {
```

Comment:
Can you add a small comment on here? exclude keys are appended to the include keys?

Reply:
Yes, in order to exclude a specific key, we have to include n-1 keys. I've added a comment similar to the if statement before this one

Reply:
Oh I see, lets say I wanted to exclude('foo.key') I would have to include `foo`

Reply:
Yup, that's it